src/buildercore/trop.py

```diff
@@ -546,6 +546,36 @@ def render_cloudfront(context, template, origin_hostname):
             SslSupportMethod='sni-only'
         )
     }
+    if context['cloudfront']['errors']:
+        props['Origins'].append(cloudfront.Origin(
+            DomainName=context['cloudfront']['errors']['domain'],
+            # TODO: constant
+            Id='errors',
+            # no advantage in using cloudfront.S3Origin for public buckets
+            CustomOriginConfig=cloudfront.CustomOrigin(
+                HTTPSPort=443,
+                OriginProtocolPolicy='https-only'
+            )
+        ))
+        props['CacheBehaviors'] = [
+            cloudfront.CacheBehavior(
+                TargetOriginId='errors',
+                ForwardedValues=cloudfront.ForwardedValues(
+                    QueryString=False
+                ),
+                PathPattern=context['cloudfront']['errors']['pattern'],
+                ViewerProtocolPolicy='allow-all',
+            ),
+        ]
+        all_codes = [400, 403, 404, 405, 414, 416, 500, 501, 502, 503, 504]
```

Comment:
Hmm, probably not `503` and `504`, Journal can produce those.

`414` probably should be too.

(Again, something that should be easier with Fastly.)